apps/src/templates/census2017/CensusForm.jsx

```diff
@@ -231,46 +273,143 @@ class CensusForm extends Component {
         </h2>
         <form id="census-form">
```

Comment:
This form is getting pretty long.  As you have noticed, it will probably be good to restructure somewhat :)